eland/operations.py

```diff
@@ -1051,52 +1051,26 @@ def describe(self, query_compiler):
                 f"Can not count field matches if size is set {size}"
             )
 
-        numeric_source_fields = query_compiler._mappings.numeric_source_fields()
-
-        # for each field we compute:
-        # count, mean, std, min, 25%, 50%, 75%, max
-        body = Query(query_params.query)
-
-        for field in numeric_source_fields:
-            body.metric_aggs("extended_stats_" + field, "extended_stats", field)
-            body.metric_aggs("percentiles_" + field, "percentiles", field)
-
-        response = query_compiler._client.search(
-            index=query_compiler._index_pattern, size=0, body=body.to_search_body()
+        df1 = self.aggs(
```

Comment:
I was carried away by implementing what pandas did, but elasticsearch and eland could do more

When implementing `quantile` along with other aggs
Pandas only used to accept quantile and give same result as of median in combination of other aggs

So, the logic is only to take 50th percentile when called with other aggs. I did the same.

```bash
          AvgTicketPrice  FlightDelayMin  dayOfWeek
max          1199.729053           360.0        6.0
min           100.020528             0.0        0.0
quantile      640.556668             0.0        3.0
median        640.556668             0.0        3.0

```
For this specific reason, I had to do a two queries.

So, do you think we should enhance the agg, by adding a `q` parameter that can accept `quantile` and its values.
And do more than Pandas.

In that case I can make change to aggregation core logic itself. (Small change though!)

And this can go in a single query.